interface/usergroup/facility_user_admin.php

```diff
@@ -140,56 +160,62 @@
             <div class="col-12">
                 <div class="page-title">
                     <h3><?php echo xlt('Edit Facility Specific User Information'); ?></h3>
+                    &nbsp;
```

Comment:
oh, oh, @tywrenn may strike this line down with a lightning bolt :)

![tenor (30)](https://user-images.githubusercontent.com/278968/85822148-08f76b80-b72f-11ea-9075-192d7c88673a.gif)

Reply:
Cleared this mess before lightning Yoda come :wink: